features/roomdetails/api/src/main/kotlin/io/element/android/features/roomdetails/api/RoomDetailsEntryPoint.kt

```diff
@@ -38,7 +38,17 @@ interface RoomDetailsEntryPoint : FeatureEntryPoint {
         data object RoomNotificationSettings : InitialTarget
     }
 
-    data class Inputs(val initialElement: InitialTarget) : NodeInputs
+    data class Params(val initialElement: InitialTarget) : NodeInputs
```

Comment:
What's the reason that led you to rename this? :)

Reply:
I changed to using the NodeBuilder pattern with `params` and `callback` as the functions in the builder as it seemed to be the predominant pattern in the codebase and thought it might bring a little more consistency to the navigation code.

Reply:
Super then :) That's exactly what I wanted to know.